activation/handler_v2_test.go

```diff
@@ -125,21 +162,45 @@ func (h *handlerMocks) expectAtxV2(atx *wire.ActivationTxV2) {
 	h.expectStoreAtxV2(atx)
 }
 
+func (h *handlerMocks) expectMergedAtxV2(
+	atx *wire.ActivationTxV2,
+	equivocationSet []types.NodeID,
+	poetLeaves []uint64,
+) {
+	h.mclock.EXPECT().CurrentLayer().Return(postGenesisEpoch.FirstLayer())
+	h.expectFetchDeps(atx)
+	h.mValidator.EXPECT().VRFNonceV2(
+		atx.SmesherID,
+		gomock.Any(),
+		atx.VRFNonce,
+		atx.TotalNumUnits(),
+	)
+	h.expectVerifyNIPoSTs(atx, equivocationSet, poetLeaves)
+	h.expectStoreAtxV2(atx)
+}
+
 func (h *v2TestHandler) createAndProcessInitial(t *testing.T, sig *signing.EdSigner) *wire.ActivationTxV2 {
 	t.Helper()
 	atx := newInitialATXv2(t, h.handlerMocks.goldenATXID)
 	atx.Sign(sig)
-	p, err := h.processInitial(atx)
+	p, err := h.processInitial(t, atx)
 	require.NoError(t, err)
 	require.Nil(t, p)
 	return atx
 }
 
-func (h *v2TestHandler) processInitial(atx *wire.ActivationTxV2) (*mwire.MalfeasanceProof, error) {
+func (h *v2TestHandler) processInitial(t *testing.T, atx *wire.ActivationTxV2) (*mwire.MalfeasanceProof, error) {
+	t.Helper()
 	h.expectInitialAtxV2(atx)
 	return h.processATX(context.Background(), peer.ID("peer"), atx, codec.MustEncode(atx), time.Now())
 }
 
+func (h *v2TestHandler) processSoloAtx(t *testing.T, atx *wire.ActivationTxV2) (*mwire.MalfeasanceProof, error) {
```

Comment:
In not the only occurrence in this file:

```suggestion
func (h *v2TestHandler) processSoloAtx(tb testing.TB, atx *wire.ActivationTxV2) (*mwire.MalfeasanceProof, error) {
```